keep/api/models/alert.py

```diff
@@ -515,6 +516,36 @@ def from_db_incident(cls, db_incident: "Incident"):
         dto._tenant_id = db_incident.tenant_id
         return dto
 
+    def to_db_incident(self) -> "Incident":
+        """Converts an IncidentDto instance to an Incident database model."""
+        from keep.api.models.db.alert import Incident
+        db_incident = Incident(
+            id=self.id,
+            user_generated_name=self.user_generated_name,
+            ai_generated_name=self.ai_generated_name,
+            user_summary=self.user_summary,
+            generated_summary=self.generated_summary,
+            assignee=self.assignee,
+            severity=self.severity.order,
+            status=self.status.value,
+            creation_time=self.creation_time or datetime.datetime.utcnow(),
+            start_time=self.start_time,
+            end_time=self.end_time,
+            last_seen_time=self.last_seen_time,
+            alerts_count=self.alerts_count,
+            affected_services=self.services,
+            sources=self.alert_sources,
+            is_predicted=self.is_predicted,
+            is_confirmed=self.is_confirmed,
+            rule_fingerprint=self.rule_fingerprint,
+            same_incident_in_the_past_id=self.same_incident_in_the_past_id,
+            merged_into_incident_id=self.merged_into_incident_id,
+            merged_by=self.merged_by,
+            merged_at=self.merged_at,
+        )
```

Comment:
wouldn't that work?:
```suggestion
        db_incident = Incident(**self.dict())
```
Maybe Incident should have a from_dto class function?

Reply:
> db_incident = Incident(**self.dict())

No, the names of some of the fields differ, eg:
```
affected_services=self.services,
sources=self.alert_sources,
```

> Maybe Incident should have a from_dto class function?

Ig we can do that.